planetiler-core/src/main/java/com/onthegomap/planetiler/stream/WriteableStreamArchive.java

```diff
@@ -34,6 +36,8 @@
  */
 abstract class WriteableStreamArchive implements WriteableTileArchive {
 
+  private final LongAdder bytesWritten = new LongAdder();
```

Comment:
Can we use `Counter.newMultiThreadCounter` here and get a new `counterForThread` from it in the factory?